packages/react-data-grid/src/common/editors/TextEditor.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useRef, useEffect } from 'react';
+
+import { EditorProps } from '../types';
+
+type TextEditorProps = Pick<EditorProps<string>, 'value' | 'onChange' | 'onCommit'>;
+
+export default function TextEditor({
+  value,
+  onChange,
+  onCommit
+}: TextEditorProps) {
+  const inputRef = useRef<HTMLInputElement>(null);
+  useEffect(() => {
+    inputRef.current?.focus();
+    inputRef.current?.select();
+  }, []);
+
+  function onKeyDown(e: React.KeyboardEvent<HTMLInputElement>) {
+    if (e.key === 'ArrowLeft' || e.key === 'ArrowRight') {
+      e.stopPropagation();
```

Comment:
I think we should just disable keyboard shortcuts while we have an editor open in rdg itself, otherwise we'll need a variation of this for every editor.